test/integration_test.go

```diff
@@ -644,6 +644,19 @@ func (s *IntegrationSuite) TestShowShardingTable() {
 	assert.NoErrorf(t, err, "show sharding table from employees error: %v", err)
 }
 
+func (s *IntegrationSuite) TestShowDatabaseRules() {
+	var (
+		db = s.DB()
+		t  = s.T()
+	)
+
+	rows, err := db.Query("show database rules from employees")
```

Comment:
should we construct some mock rule to validate the result set?

Reply:
Need to add db rule, which may affect other ut